### PR TITLE
ICU-21581 Promoted @draft 68 APIs to stable

### PR DESCRIPTION
**Pull request by richgillam**
##### Checklist

- [x] Required: Issue filed: https://unicode-org.atlassian.net/browse/ICU-21581
- [x] Required: The PR title must be prefixed with a JIRA Issue number. 
- [x] Required: The PR description must include the link to the Jira Issue, for example by completing the URL in the first checklist item
- [ ] Required: Each commit message must be prefixed with a JIRA Issue number. 
- [x] Issue accepted (done by Technical Committee after discussion)
- [ ] Tests included, if applicable
- [ ] API docs and/or User Guide docs changed or added, if applicable

### CODE REVIEW

File: icu4c/source/i18n/unicode/numberformatter.h [hunk 1]
```diff
@@ -2725,10 +2722,11 @@ class U_I18N_API FormattedNumber : public UMemory, public FormattedValue {
      * as "foot-and-inch" or "hour-and-minute-and-second".
      *
      * @return `MeasureUnit`.
-     * @draft ICU 68
+     * @stable ICU 68
      */
     MeasureUnit getOutputUnit(UErrorCode& status) const;
 
+#ifndef U_HIDE_DRAFT_API
```
Comment (sffc) [inline icu4c/source/i18n/unicode/numberformatter.h:2729]: Should this one be U_HIDE_INTERNAL_API instead of U_HIDE_DRAFT_API ?

Comment (richgillam) [reply icu4c/source/i18n/unicode/numberformatter.h:2729]: Oof.  You're right; I included one method I didn't mean to include.  Fix forthcoming...

Comment (richgillam) [reply icu4c/source/i18n/unicode/numberformatter.h:2729]: Sorry; I was looking at the wrong branch.  But yes, after looking at the right branch, I agree-- it looks like this should have been inside U_HIDT_DRAFT_API.  Fix forthcoming...

**Review by sffc — DISMISSED** [review]

**Comment by sffc** [comment]
LGTM please squash

**Review by markusicu — APPROVED** [review]
lgtm tnx

**Pull request merged**